packages/react-router/src/components/FocusContext/FocusContext.tsx

```diff
@@ -1,11 +1,11 @@
-import React, {useRef, useEffect, ReactNode, createRef} from 'react';
+import React, {useRef, useEffect, ReactNode} from 'react';
 import {FocusContext as Context} from '../../context';
 import {useCurrentUrl} from '../../hooks';
 import {Focusable} from '../../types';
 
 export function FocusContext({children}: {children: ReactNode}) {
   const currentUrl = useCurrentUrl();
-  const focusRef = createRef<Focusable>();
+  const focusRef = useRef<Focusable>();
```

Comment:
`createRef` returns a _RefObject_, while `useRef` returns a _MutableRefObject`. From what I was reading, I should have been using `useRef` instead.

Reply:
I don't know how I missed this :/